### PR TITLE
React- Minor Refactor

### PR DESCRIPTION
**Pull request by drod1000**
@jdconrad89 @mollybrown 
Worked on this branch with @Laszlo-JFLMTCO while making cards for the next sprint. We figured these were minor changes that were needed so we figured it would be best to just take care of them now. 
Closes #143346705
@neight-allen

### CODE REVIEW

File: app/assets/javascripts/components/alumni/_main.js.jsx [hunk 1]
```diff
@@ -10,30 +10,20 @@ var Main = React.createClass({
...
-          <MainSearch prepNewPortfolios={this.prepNewPortfolios} port_asc={this.props.port_asc} port_desc={this.props.port_desc} portfolios={this.props.portfolios} locations={this.props.locations}/>
-          <UserCard portfolios={newPortfolios} avatars={this.props.avatars} bios={this.props.bios} locations={this.props.locations}/>
-          <Footer/>
-        </div>
-      )
+    if(this.props.user != null) {
+      var navbar = <SignedInNavbar user={this.props.user}/>
+    }
+    else {
+      var navbar = <GuestNavbar/>
```
Comment (jdconrad89) [inline app/assets/javascripts/components/alumni/_main.js.jsx:17]: 👍

**Comment by jdconrad89** [comment]
@drod1000 Looks good to me! Thanks or refactoring this! I'm good to merge this if no one has any objections

File: app/assets/javascripts/components/dashboard/_dashboard.js.jsx [hunk 2]
```diff
@@ -1,18 +1,25 @@
...
+              }
 
-              { projects != null && projects.length > 0 ?  <DashboardProject projectAvatars={this.props.project_avatars} user={user} projects={projects}/>: <NoDashboardProjects user={user} portfolio={portfolio}/>}
+              { this.props.projects != null && this.props.projects.length > 0
+                ?
+                <DashboardProject projectAvatars={this.props.project_avatars} user={this.props.user} projects={this.props.projects}/>
+                :
+                <NoDashboardProjects user={this.props.user} portfolio={this.props.portfolio}/>
+              }
             </div>
```
Comment (mollybrown) [inline app/assets/javascripts/components/dashboard/_dashboard.js.jsx:23]: Not a giant fan of the ternary, but this is a much better way of displaying it that before, so good job on the refactor.

Comment (neight-allen) [reply app/assets/javascripts/components/dashboard/_dashboard.js.jsx:23]: Agreed on ternary not being great. Lovisa uses them in her creact tutorial, and I think that's where Turing students get it from, but for multi-line stuff, it's confusing.

File: app/assets/javascripts/components/alumni/_main.js.jsx (hunk 1 above)
Comment (neight-allen) [reply app/assets/javascripts/components/alumni/_main.js.jsx:17]: This is kind of just a style thing, but if I'm just testing for the existence of something, I'll use `if(!this.props.user)`

I also try to make all of my conditionals positive, so I'd probably further refactor it to be `if(this.props.user)` and swap my if/else.

**Review by neight-allen — COMMENTED** [review]
I know I'm too late, but left a couple style thoughts @jdconrad89 @mollybrown @drod1000 @Laszlo-JFLMTCO

**Pull request merged**